telegram/chat.py

```diff
@@ -364,7 +459,7 @@ def send_video_note(self, *args, **kwargs):
     def send_voice(self, *args, **kwargs):
         """Shortcut for::
 
-            bot.send_voice(Chat.id, *args, **kwargs)
+            bot.send_voice(update.effective_chat.id, *args, **kwargs)
 
         Where Chat is the current instance.
```

Comment:
'Where Chat is the current instance' should be removed everywhere since it was changed to `update.effective_chat.id` instead of `Chat.id` right?

Reply:
Same thing in `user.py` too

Reply:
Yeah, maybe you're right :) If I remember, I'll add it in #2008. If you want to, you can also PR for that. Just leave a comment here in that case, so we don't do the work twice

Reply:
Edit: Done